section-2/exercises/sviatoslavYurkivSection2.js

```diff
@@ -0,0 +1,44 @@
+
+//Exercise 1:
+const currentTemperature = 0;
+const isRaining = true;
+const cityName = "City";
+
+//Exercise 2: will be an error SyntaxError: Missing initializer in const declaration
+
+// Exercise 3:
+let messageOne = 'Message 1';
+const messageTwo = messageOne
+console.log(messageOne)
+console.log(messageTwo)
+
+// Exercise 4: What will show the "Igor" name for userName and secondUserName
+const userName = 'Igor';
+const secondUserName = userName;
+console.log(userName)
+console.log(secondUserName);
+
+//Exercise 5:
+
+const text = "Hello World!";
+const lowerCaseText = text.toLocaleLowerCase()
```

Comment:
toLocaleLowerCase() method also ok here but, I would recommend to use toLowerCase() method